### ISSUE TITLE
TST: setup.py bdist_wheel error for numpy-dev

### ISSUE THREAD

**Issue by pllim** [astropy/astropy#8318]
https://travis-ci.org/astropy/astropy/jobs/473606827

Error started here and continued on for a while:
```
Collecting numpy from git+https://github.com/numpy/numpy.git#egg=numpy
  Cloning https://github.com/numpy/numpy.git to /tmp/pip-install-uu21f19y/numpy
Building wheels for collected packages: numpy
  Running setup.py bdist_wheel for numpy ... error
  Complete output from command /home/travis/miniconda/envs/test/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-uu21f19y/numpy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/pip-wheel-a_dluvtg --python-tag cp37:
  Running from numpy source directory.
  Cythonizing sources
  Processing numpy/random/mtrand/randint_helpers.pxi.in
  Processing numpy/random/mtrand/mtrand.pyx
  blas_opt_info:
  blas_mkl_info:
  customize UnixCCompiler
    libraries mkl_rt not found in ['/home/travis/miniconda/envs/test/lib', '/usr/local/lib', '/usr/lib', '/usr/lib/x86_64-linux-gnu']
    NOT AVAILABLE

...
```

cc @mhvk and @bsipocz

**Comment by mhvk** [comment]
This seems like a conda problem (which I know nothing about; indeed, this failure seems one more reason to stick with Debian...)

**Comment by saimn** [comment]
Since the errors come from `numpy/fft/pocketfft.c` and the use of c89, I actually think that this is related to a numpy change. https://github.com/numpy/numpy/pull/12610 (*C99 is the
standard for NumPy >= 1.17*) lead to https://github.com/numpy/numpy/pull/11888: integration of a new library for fft, compatible with c99 only.
So I guess that we need to use c99 for the numpy-dev build, unless they plan to keep a c89 mode which is not working. But using c99 is probably the best way to go.

**Comment by pllim** [comment]
@saimn , do you want to open an issue with `numpy`? Thank you! 🙏

**Comment by pllim** [comment]
Thanks to @rgommers , this would be fixed once astropy/ci-helpers#368 is merged.

**Comment by saimn** [comment]
Thanks for taking care of this @pllim :)
